scale/ingest/strike/monitors/dir_monitor.py

```diff
@@ -78,13 +81,11 @@ def stop(self):
     def validate_configuration(self, configuration):
         """See :meth:`ingest.strike.monitors.monitor.Monitor.validate_configuration`
         """
-
-        if 'transfer_suffix' not in configuration:
-            raise InvalidMonitorConfiguration('transfer_suffix is required for dir-watcher monitor')
-        if not isinstance(configuration['transfer_suffix'], basestring):
-            raise InvalidMonitorConfiguration('transfer_suffix must be a string')
-        if not configuration['transfer_suffix']:
-            raise InvalidMonitorConfiguration('transfer_suffix must be a non-empty string')
+        if 'transfer_suffix' in configuration:
```

Comment:
Is there any way to return a warning if transfer_suffix isn't defined? Something that will return saying something like "No transfer suffix defined, '_tmp' will be used"?